docs/tutorials/ml_inference/semantic_search/bedrock_titan_multi-modal_embedding_model.md

```diff
@@ -0,0 +1,644 @@
+        "multimodal_embeddings": [
+            0.016407186,...],
+        "name": "Time Sqaure",
+        "category": "Sqaure"
+    }
+}
+```
+Load a example doc no text nor image.
+
+```
+PUT test-index-area/_doc/3
+{
+   "category": "Bridge"
+}
+```
+Check doc/3, when no matching input mapping is provided, the processor is skipped and still ingest the document. 
+
+```
+GET test-index-area/_doc/3
+```
+Sample Response
+```
+{
+    "_index": "test-index-area",
+    "_id": "3",
+    "_version": 1,
+    "
```

Comment:
Looks complex configuration ? Why need three `ml_inference` processor here while the ingest pipeline just need one ?

Reply:
update using optional mapping